cmake/ConkyPlatformChecks.cmake

```diff
@@ -256,7 +261,12 @@ if(BUILD_NCURSES)
   endif(NOT CURSES_FOUND)
 
   set(conky_libs ${conky_libs} ${CURSES_LIBRARIES})
-  set(conky_includes ${conky_includes} ${CURSES_INCLUDE_DIR})
+  if(OS_NETBSD)
+    set(conky_includes ${conky_includes} /usr/pkg/include)
+    set(conky_includes ${conky_includes} /usr/pkg/include/ncurses)
+  else(OS_NETBSD)
+    set(conky_includes ${conky_includes} ${CURSES_INCLUDE_DIR})
+  endif(OS_NETBSD)
```

Comment:
This whole `BUILD_NCURSES` block should really do what [this patch for weechat on NetBSD does](http://cvsweb.netbsd.org/bsdweb.cgi/pkgsrc/chat/weechat/patches/patch-cmake_FindNcurses.cmake?rev=1.1&content-type=text/x-cvsweb-markup) - check for both `ncurses` and `curses`.